README.md

```diff
@@ -45,6 +45,10 @@ On commit to master Jenkins will build both `dotnetcore` and `dotnetcore-develop
 
 This image uses the [Defra Docker Shared Jenkins library](https://github.com/DEFRA/defra-docker-jenkins) to abstract pipeline complexity from repository.  See repository for further usage details.
 
+## Image vulnerability scanning
+
+The repository runs a nightly Anchore Engine scan, and will scan images on push. For more details see [Image Scanning](IMAGE_SCANNING.md).
+
```

Comment:
Is it worth putting something in here just to say _why_ we're doing this - basically that there isn't an automated scanning tool that we can just hook up to the source code that will identify vulnerabilities so we've got to scan the actual image itself?